static/app/components/alerts/snoozeAlert.tsx

```diff
@@ -106,20 +115,24 @@ function SnoozeAlert({isSnoozed, onSnooze, projectSlug, ruleId, hasAccess}: Prop
 
   useEffect(() => {
     if (location.query.mute === '1' && !isSnoozed) {
-      handleMute('me', true);
+      const target =
+        ruleActionCategory !== RuleActionsCategories.AllDefault ? `everyone` : `me`;
+      handleMute(target, true);
     }
-  }, [location.query, isSnoozed, handleMute]);
+  }, [location.query, isSnoozed, handleMute, ruleActionCategory]);
 
   const dropdownItems: MenuItemProps[] = [
     {
       key: 'me',
       label: t('Mute for me'),
       onAction: () => handleMute('me'),
+      hidden: ruleActionCategory !== RuleActionsCategories.SomeDefault,
```

Comment:
Would love a comment over these `hidden` props explaining why this option should be disabled for these cases